tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+
+    <html>
+    <head><title>404 Not Found</title></head>
+    <body>
+    <center><h1>404 Not Found</h1></center>
+    <hr><center>nginx</center>
+    </body>
+    </html>
+    ```
+
+10. With `dig` command, you can check that your DNS record is resolving to the IPs of all of the healthy load balancers
+    ```
+    dig @ns1.dom.scw.cloud ingress.scw.your-domain.tld
+
+    ...
+    ;; ANSWER SECTION:
+    ingress.scw.your-domain.tld. 60 IN    A       195.154.68.56
+    ingress.scw.your-domain.tld. 60 IN    A       51.159.85.96
+    ...
+    ```
+
+Your multi-AZ Ingress Nginx Controller should now available on `http://ingress.scw.your-domain.tld`
+
+To automatically manage TLS certificates for your HTTPS ingresses, you can [deploy the cert-manager](https://cert-manager.io/).
+
+## Elasticsearch/Kibana as a stateful multi-AZ application
+
+In this last section, you will deploy a multi-AZ Elasticsearch and a Kibana instance to query it. It will allow you to work with the stateful constraints imposed by the use of persistent volumes in a multi-AZ architecture. As a prerequisite, you need to have a default `scw-bssd` storage class with a `volumeBindingMode` set to `WaitForFirstConsumer`. You can check your cluster with the `kubectl` command below. Previous clusters were provisioned with the binding mode set to `Immediate`, so if you are in this case you need to update your cluster to the latest patch version (kubernetes >=1.24.17, >=1.25.13, >=1.26.8, >=1.27.5 or >=1.28.1). The storage class will be automatically updated to the new binding mode during the update.
+    ```
+    kubectl get storageclasses.storage.k8s.io
+
+    NAME                 PROVISIONER        RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+    scw-bssd (default)   csi.scaleway.com   Delete          WaitForFirstConsumer   true                   131m
+    scw-bssd-retain      csi.scaleway.com   Retain          WaitForFirstConsumer   true                   131m
+    ```
+
+The use of a storage class with `volumeBindingMode` option set to `WaitForFirstConsumer` is essential to deploy a multi-az application using persistent volumes. This option is necessary to make sure that the volume is created only when the pod is scheduled on a node. This is important because the volume is restricted to a specific AZ and ca not be used or moved to another AZ. So if we create the volume before the pod is scheduled, the CSI driver will not know on which AZ it will be used, so it will create the volume on a random AZ, and at the binding time, if the pod trying to attach the volume is scheduled on another AZ, it will fail to attach the volume and the pod will fail to start. With the `WaitForFirstConsumer` option, the volume is created only when the pod is scheduled on a node, so the volume is created on the AZ of the node where the pod is scheduled. Since pods are spread across the different AZs, the volumes will also be spread across the different AZs.
```

Comment:
This is a very long chunk of text.
I suggest to jump a space or two before starting 
- This is important
- With the `WaitForFirstConsumer` option...
Better for the eyes, better read.

Reply:
```suggestion
It is important to use a storage class with the `volumeBindingMode` option set to `WaitForFirstConsumer` when deploying a multi-AZ application with persistent volumes. 

This option ensures that the volume is created only when the pod is scheduled on a node, addressing the specific AZ requirement. Creating the volume before scheduling the pod could result in the volume being placed randomly in an AZ, potentially causing attachment failures if the pod is scheduled in a different AZ. 

With the `WaitForFirstConsumer` option, volumes are created on the same AZ as the node where the pod is scheduled, effectively spreading volumes across different AZs as pods are distributed.
```

@Nox-404 wdyt?

Reply:
LGTM, it's clearly more readable